src/styles/main.scss

```diff
@@ -1,3 +1,40 @@
+@import "utils/variables";
+
 body {
-  margin: 0;
+  font-size: 30px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-top: 50px;
+}
+
+.calendar {
+  align-items: center;
+  gap: $gap-size;
+  flex-wrap: wrap;
+  display: flex;
+  width: 720px;
+  padding: 10px;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: $day-block-size;
+    width: $day-block-size;
+    background-color: #eee;
+    border: 1px solid black ;
+    transition-duration: 500ms;
```

Comment:
```suggestion
    transition-duration: transform 500ms, background-color 500ms;
```
It is a good tone to specify the exact properties.
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/77523e3d-aece-45a8-8e7d-906a23fb95c3)
Also, let's create a variable for transition-duration. Same with colors